docker-compose.yml

```diff
@@ -1,15 +1,18 @@
 services:
   postgres:
     image: postgres:16
+    container_name: hostedgpt_postgres
     restart: always
     environment:
-      POSTGRES_USER: app
-      POSTGRES_DB: app_development
```

Comment:
rails will create the database so we don't need to make the postgres init process do it